rat/parser.py

```diff
@@ -641,7 +675,7 @@ def statement(self):
                 try:
                     return_statement = Distributions.generate(lhs, expressions, distribution)
                 except TypeCheckError as e:
-                    raise ParseError(str(e), range)
+                    raise ParseError(str(e), lhs.range)
```

Comment:
I think the range here should probably come from the left hand side of:

```
self.expect_token(Identifier, Distributions.names)
```

expect_token returns the token now I think